main.cpp

```diff
@@ -52,85 +74,88 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun:car
+//  action 1:car moves forward
+car.moveForward();  
 
-//  action 2:
+//  action 2:car moves backward
+car.moveBackward();
 
-//  action 3:
+//  action 3:car stops moving
+car.stop();
  
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  10)
-//  Noun:
+//  Noun:foot  
+//  action 1:foot moving in any direction
+foot.move();
+//  action 2:rotate foot to the left
+foot.rotateLeft()
+//  action 3:stomps foot
+foot.stomp();
+//  3)  
+//  Noun: cat
+//  action 1:cat meows
+cat.meow();
+//  action 2:cat scratches post
+cat.scratchPost();
+//  action 3:cat naps
+cat.nap();
+//  4) 
+//  Noun: ball
+//  action 1:ball bounces
+ball.bounce(); 
+//  action 2:ball strikes an object
+ball.strikesObject();
+//  action 3:ball stops moving
+ball.stop()
+//  5) 
+//  Noun:game
+//  action 1:game starts
+game.start();
+//  action 2:game pauses playback
+game.pauses();
+//  action 3:game finishes
+game.finish();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.

Reply:
replaced 'game' with 'hardDisk' and changed actions and English descriptions